### DISCUSSION TITLE
Forgot export for StaticBlock in @types/estree/index

### DISCUSSION THREAD

**Discussion by imishurovskiy** [DefinitelyTyped/DefinitelyTyped#64376] [Issues with a @types package]
Error: node_modules/@types/eslint/index.d.ts:494:38 - error TS2694: Namespace '"node_modules/@types/estree/index"' has no exported member 'StaticBlock'.

[a comment by an automated account is not shown]

**Reply by Kirperepolov** [reply]
Missing StaticBlock  inside [@types/estree/index].

Seems like after [this change ](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64317/files#diff-c679de750c8bcef1dcbf631a61258c78c66abb2be8b49e10331b5b93de649743) npm scripts fail with the mentioned error:
```
../../node_modules/@types/eslint/index.d.ts:494:38 - error TS2694: Namespace '"C:/Users/kyrylo.o.perepolov/Dev/CASE_old/npm.repo/node_modules/@types/estree/index"' has no exported member 'StaticBlock'.

494         StaticBlock?: ((node: ESTree.StaticBlock & NodeParentExtension) => void) | undefined;
```

**Comment by Kirperepolov** [comment]
try to define the freshest dependency manually:
`npm install --save-dev @types/estree@^1.0.0`.
seems like your eslint is one of the updated versions, while @types/estree in package-lock is outdated (I had 0.0.50 version) and make the dependencies inconsistent.
